docs/dev/local_build_and_testing.md

```diff
@@ -0,0 +1,37 @@
+## Dev Database
+Use docker to launch a local postgres DB, flyway used to install schema. Steps:
+```bash
+cd lobby-db
+# build docker only needs to be run once
+./build_docker
+./run_docker
+# wait about 3 or 5 seconds for database to start
+./run_flyway
+```
+Docker for Mac can be obtained at: https://store.docker.com/editions/community/docker-ce-desktop-mac
```

Comment:
Actually this link doesn't help for users that don't have a docker account. I don't know if you had to deal with this before but I certainly had to:
As described in this issue: https://github.com/docker/docker.github.io/issues/6910 direct links still exists, but are sort of hidden in the release notes: https://docs.docker.com/docker-for-mac/release-notes/

But for simplicity I'd recommend installing docker via `brew cask install docker` if you have brew and brew cask installed. (Which as a developer you probably should)

Reply:
We did not have much information on the docker page, this seemed to have been the most useful. The update here is simply dropping that one line of info to the main page and then deleting the rest of the docker page. It's been some time since I installed docker on a mac, I can't speak to the best way of doing it for now. It could make sense to delete this link entirely if it is not useful and allow the next person to install on mac to update the notes.